packages/itwinui-css/src/input/classes.scss

```diff
@@ -21,3 +25,33 @@
 .iui-end-icon {
   @include iui-end-icon;
 }
+
+.iui-input-flex-container {
+  @include iui-input-flex-container;
+
+  > input[type='search'] {
+    @include iui-search-box;
+  }
+
+  &[data-iui-size='small'] {
```

Comment:
could we handle sizing automatically if the user passes, for example, `<Input size='small'>` as child?

Reply:
Created a context for sizing: when user passes <Searchbox size='small' /> - buttons, inputs and icons are being rendered small.